### ISSUE TITLE
loadTimeout is fired even if it already successfully loaded the ad

### ISSUE THREAD

**Issue by nemoryoliver** [bdlukaa/native_admob_flutter#19]
```
BannerAd(
      unitId: admobId,
      controller: _nafController,
      size: naf.BannerSize.ADAPTIVE,
      loadTimeout: Duration(seconds: 30),
    )
```

after 30 seconds, even successfully loaded, the controller fires a timeout: loadAdFailed event

**Comment by bdlukaa** [comment]
Will be fixed in the next version!

**Comment by bdlukaa** [comment]
Fixed in [1.0.0](https://pub.dev/packages/native_admob_flutter)
